src/main/java/picard/nio/PathHelper.java

```diff
@@ -0,0 +1,42 @@
+package picard.nio;
+
+import picard.cmdline.ClassFinder;
+
+/**
+ * Created by farjoun on 11/13/17.
+ */
+public class PathHelper {
+
+    private PathHelper() {}
+
+    public static boolean isHasGoogle() {
```

Comment:
isHas? That is all.

Reply:
I forsee a day in which we will have 

isHasGoogle(), isHasAws(), isHasAzure() etc....

Reply:
that said, I agree that the name is somewhat awkward...but I think it's good to keep google, or GCS in the name.

Reply:
how about `supports()`?  (see my comments about not making the API google specific)